nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -25,7 +25,7 @@ import (
 // +kubebuilder:object:root=true
 
 // NodeNetworkConfig is the Schema for the nodenetworkconfigs API
-// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:resource:scope=Namespaced
```

Comment:
just a heads up with this change, if you have a cluster laying around with the crd already loaded in (like me with my local minikube for example) you'll have to `kubectl delete crd acn.azure.com/NodeNetworkConfigs` or similar, since you can't change a CRD from cluster-scope to namespace-scope.

Reply:
Thanks, I'm going to review your PR btw soon